app/lib/downloadFile.js

```diff
@@ -0,0 +1,25 @@
+const log = require('./logger');
+const apiRequest = require('./apiRequest');
+const fsHelper = require('./fsHelper');
+const constants = require('./constants');
+
+const POMI_FILE = constants.POMI_FILE;
+const POMI_DOWNLOAD_LOCATION = constants.POMI_DOWNLOAD_LOCATION;
+
+function downloadFile() {
+  return new Promise((resolve, reject) => {
```

Comment:
The API request already returns a promise, the wrapping promise isn't required and the body could be written as 

```
 log.time('Downloading the POMI file took');
 return apiRequest(POMI_DOWNLOAD_LOCATION)
   .then((data) => {
     log.timeEnd('Downloading the POMI file took');
     fsHelper.saveFileSync(data, POMI_FILE);
   });
```